api/desecapi/serializers.py

```diff
@@ -172,7 +172,7 @@ class RRsetSerializer(ConditionalExistenceModelSerializer):
 
     class Meta:
         model = RRset
-        fields = ('domain', 'subname', 'name', 'records', 'ttl', 'type',)
+        fields = ('created', 'domain', 'subname', 'name', 'records', 'ttl', 'type',)
```

Comment:
This should also be added to the e2e schema test in `test/e2e/schemas.js`

Reply:
good catch! done